.travis.yml

```diff
@@ -21,4 +21,4 @@ before_install:
 script:
   # PATH is not set under sudo, then we set up path mannually.
   # Ref https://unix.stackexchange.com/questions/83191/how-to-make-sudo-preserve-path
```

Comment:
Could we move the comment to the Makefile?

Reply:
Yes, I have update it.